### PR TITLE
DOC-6155: 4.11.1 release notes and checksums updates

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ weight: 10
 
 # Corda Enterprise Edition 4.11 release notes
 
+## Corda Enterprise Edition 4.11.1 release notes
+
+Interoperability fix between 4.11 and pre-4.11 nodes when sending/fetching transactions for new data type: `TRANSACTION_RECOVERY`.
```
Comment (adelel1) [inline content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md:22]: For consistency, can you use same format as 4.10.3, i.e. Upgrade Recommendation, Fixed Issues title etc, you can use same text as 4.10.3 for upgrade recommendation.

File: content/en/platform/corda/4.11/community/release-notes.md [hunk 2]
```diff
@@ -20,6 +20,12 @@ tags:
 
 # Corda Community Edition 4.11 release notes
 
+## Corda Community Edition 4.11.1 release notes
+
+Interoperability fix between 4.11 and pre-4.11 nodes when sending/fetching transactions for new data type: `TRANSACTION_RECOVERY`.
```
Comment (adelel1) [inline content/en/platform/corda/4.11/community/release-notes.md:25]: For consistency can you use same format as Community release notes 4.10 https://docs.r3.com/en/platform/corda/4.10/community/release-notes.html

File: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md]: Updated

File: content/en/platform/corda/4.11/community/release-notes.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/community/release-notes.md]: Updated

**Review by adelel1 — APPROVED** [review]

**Review by ronanbrowne — APPROVED** [review]

**Pull request merged**